Source/WebKit/UIProcess/win/PageClientImpl.cpp

```diff
@@ -51,7 +51,9 @@ PageClientImpl::PageClientImpl(WebView& view)
 std::unique_ptr<DrawingAreaProxy> PageClientImpl::createDrawingAreaProxy(WebProcessProxy& process)
 {
 #if USE(GRAPHICS_LAYER_WC)
-    if (m_view.page()->preferences().useGPUProcessForWebGLEnabled())
+    if (m_view.page()->preferences().useGPUProcessForWebGLEnabled() ||
+        m_view.page()->preferences().useGPUProcessForDOMRenderingEnabled() ||
+        m_view.page()->preferences().useGPUProcessForDOMRenderingEnabled())
```

Comment:
WinCairo uses DrawingAreaProxyWC if useGPUProcessForWebGLEnabled.
PlayStation doesn't support WebGL. It should use DrawingAreaProxyWC unconditinally.
>if USE(GRAPHICS_LAYER_WC)
>if ENABLE(WEBGL)
>    bool usesWC = m_view.page()->preferences().useGPUProcessForWebGLEnabled();
>#else
>   bool usesWC = false
>#endif
>   if (usesWC)